banjo/conf/schema.xml

```diff
@@ -216,6 +216,7 @@
    <!-- link copy to work -->
    <field name="work_id" type="long" indexed="true" stored="true"/>
    <field name="work_pi" type="string" indexed="true" stored="true"/>
+   <field name="work_alias_pi" type="text_general" indexed="true" stored="true" multiValued="true"/>

```

Comment:
@danielbaker 
http://stackoverflow.com/questions/7175619/apache-solr-string-or-text
String stores a word/sentence as an exact string without performing tokenization etc. Commonly useful for storing exact matches, e.g, for facetting.
Text typically performs tokenization, and secondary processing (such as lower-casing etc.). Useful for all scenarios when we want to match part of a sentence.